src/release/breaking-changes/editable-text-scroll-into-view.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Updated EditableText scroll into view behavior
+description: >
+  Improve EditableText selection scroll into view behavior to always
+  use the current selection extent.
+---
+
+## Summary
+
+`Editable.onCaretChanged` callback is removed and with this change,
+`EditableText` behavior with regard to scrolling the selection into view
+changes.
+
+## Context
+
+To date, `EditableText` has used multiple mechanisms ensuring the selection
+extent or caret is scrolled into view on updates. 
```

Comment:
Not sure what you mean by "on updates".

```suggestion
Previously, `EditableText` used multiple mechanisms to determine the
extent of the selection, or to ensure that the caret is scrolled into view.
```

Reply:
Thanks for the edits. Yes, this is a bit convoluted: what I meant were the mechanisms used on scrolling into view (which happens on user updates) either of:
- selection extent (non-collapsed selection)
- caret position (collapsed selection).

Let me reword as:

> Previously, upon scrolling into view to show user updates, `EditableText` used multiple mechanisms to determine the
> extent of the selection or the caret location.